### PR TITLE
Add import support to consul_config_entry

### PR DESCRIPTION
**Pull request by remilapeyre**
Related to https://github.com/hashicorp/terraform-provider-consul/issues/316

### CODE REVIEW

File: consul/resource_consul_config_entry.go [hunk 1]
```diff
@@ -16,6 +16,38 @@ func resourceConsulConfigEntry() *schema.Resource {
...
+				case 4:
+					kind = parts[0]
+					name = parts[1]
+					partition = parts[2]
+					namespace = parts[3]
+				default:
+					return nil, fmt.Errorf(`expected path of the form "<kind>/<name>" or "<kind>/<name>/<partition>/<namespace>"`)
+				}
+
+				d.SetId(fmt.Sprintf("%s-%s", kind, name))
```
Comment (dhiaayachi) [inline consul/resource_consul_config_entry.go:36]: Shouldn't partition and namespace be added to the ID to ensure uniqueness?

File: templates/resources/config_entry.md [hunk 2]
```diff
@@ -280,3 +280,15 @@ The following attributes are exported:
...
 
 * `config_json` - A map of configuration values.
+
+
+## Import
+
+`consul_config_entry` can be imported using the syntax `<kind>/<name>` if the
+config entry is in the default partition and default namespace, or
+`<kind>/<name>/<partition>/<namespace>` for config entries in a non-default
+partition or namespace:
```
Comment (dhiaayachi) [inline templates/resources/config_entry.md:290]: What about providing `foo/bar/default/default` is it equivalent to `foo/bar`?

**Review by dhiaayachi — COMMENTED** [review]
👋 @remilapeyre 
I had few questions about that one, specifically around partition and namespace handling.

File: templates/resources/config_entry.md (hunk 2 above)
Comment (remilapeyre) [reply templates/resources/config_entry.md:290]: Yes it Is equivalent. Looking at it now it seems to me like `<partition>/<namespace>/<kind>/<name>` might make more sense here.

File: consul/resource_consul_config_entry.go (hunk 1 above)
Comment (remilapeyre) [reply consul/resource_consul_config_entry.go:36]: The ID does not need to be unique. For config entries there is no ID that exist so we can use any placeholder there. We have other resources that not have a unique ID either, there is some examples at: https://github.com/hashicorp/terraform-provider-consul/pull/318#discussion_r1046452455

File: templates/resources/config_entry.md (hunk 2 above)
Comment (dhiaayachi) [reply templates/resources/config_entry.md:290]: I agree, that would be more inline with consul annotations around namespace and partition.

Comment (remilapeyre) [reply templates/resources/config_entry.md]: Done!

**Pull request merged**